clang/test/CXX/drs/dr0xx.cpp

```diff
@@ -1414,7 +1414,7 @@ namespace dr96 { // dr96: no
     // FIXME: This is ill-formed, because 'f' is not a template-id and does not
     // name a class template.
     // FIXME: What about alias templates?
```

Comment:
Indeed, what about alias templates? My guess is that new rules work for them as well: no template argument list — no `template`.

Reply:
The use of 'template' before an alias template is [deprecated](https://eel.is/c++draft/depr.template.template), along with before a class template. I wasn't sure how to interpret this for clang, they aren't errors yet?

Reply:
The alias template case was aligned with the class-template case as part of [CWG1710](https://cplusplus.github.io/CWG/issues/1710.html).